lib/components/atoms/Button/Button.style.js

```diff
@@ -25,6 +25,21 @@ const ButtonStyles = css`
     outline: none;
   }
 
+  &.circle {
+    border-radius: 50%;
+  }
+
+  &.dots {
```

Comment:
This should be within the carousel, since this will probably never be needed outside carousel.